server/routes.ts

```diff
@@ -0,0 +1,49 @@
+import { Router } from "express";
+
+type NavBarEntry = { path: string; id: string; caption: string };
+
+export let navigation_bar: Array<NavBarEntry> = [
+  {
+    path: "/",
+    id: "models",
+    caption: "Models"
+  },
+  {
+    path: "/case-map",
+    id: "case-map",
+    caption: "Case map"
+  },
+  {
+    path: "/containment",
+    id: "mitigation",
+    caption: "Mitigation"
+  },
+  {
+    path: "/request-calculation",
+    id: "request-model",
+    caption: "Request model"
+  },
+  {
+    path: "/about",
+    id: "about",
+    caption: "About"
+  }
+];
+
+export let router = Router();
+
+// serve the main model visualization
+router.get("/", (req, res) => res.render("model.html"));
+
+router.get("/case-map", (req, res) => res.render("case-map.html"));
+
+router.get("/request-calculation", (req, res) =>
+  res.render("request-calculation.html", { message: "Please provide data" })
+);
+
+router.get("/about", (req, res) => res.render("about.html"));
+
+// Not sure why, but we had a lot of 404 in access log for this URL
+router.get("/containment.", (req, res) => res.redirect("/containment"));
```

Comment:
just FYI, maybe it's not a problem anymore, would be worth to add some  logging for when this redirect happens to know if we could remove it.

Reply:
I think it was because of
> The menu item linked to `/containments` instead of `/containment`

Reply:
Also: we already log urls

Reply:
I am not sure what you mean, but in the past, it simply seemed that someone somewhere shared a link with the end `dot` in it. I saw that in the access logs and rather added a redirect to `containment`.

Reply:
Hm, I think I assumed it was somehow a regex dot pattern because the website is also mysteriously redirecting `/containment_` -> `/containment` for all chars `_`.

Reply:
You are actually right! :hushed: I didn't know that when I added it, I really thought it's just a literal dot :hushed: ... But fastapi apparently treat it as regex..